packages/react-query/package.json

```diff
@@ -42,7 +42,7 @@
     "src"
   ],
   "bin": {
-    "suspensive-react-query-switch": "dist/scripts/suspensive-react-query-switch.cjs"
+    "suspensive-react-query": "dist/scripts/cli.cjs"
```

Comment:
How about providing 2 way? in cli world, kind of shorthand is usual.

```suggestion
    "suspensive-react-query": "dist/scripts/cli.cjs",
    "srq": "dist/scripts/cli.cjs"
```

Reply:
I agree, I completely forgot about that!